src/Component/Detail/DetailButton.js

```diff
@@ -0,0 +1,36 @@
+import Component from '../../core/Component.js';
+import { delegate } from '../../util/helpers.js';
+
+export default class DetailButton extends Component {
+  constructor($element, props) {
+    super($element, props);
+    this.addTeamMember = this.props.addTeamMember;
+    this.render();
+  }
+
+  handleClick() {
+    const name = prompt('새로운 팀원 이름을 입력해주세요.');
+    if (name === null || name.length <= 0) return;
```

Comment:
```suggestion
    if (!name) return;
```
이렇게 하면 조건 하나에 처리가 가능할 것같아요!

Reply:
매 주마다 들었던 피드백인것 같은데 매번 습관적으로 이렇게... 😞 코드를 계속 보면서 논리적으로 생각해봐야겠네요. 감사합니다!